tgbotapi.core/src/commonMain/kotlin/dev/inmo/tgbotapi/types/chat/Extended.kt

```diff
@@ -58,8 +58,8 @@ data class ExtendedChannelChatImpl(
     override val newMembersSeeHistory: Boolean = false,
     @SerialName(maxReactionCountField)
     override val maxReactionsCount: Int = 3,
-    @SerialName(canSendGiftsField)
-    override val canReceiveGifts: Boolean = false,
+    @SerialName(acceptedGiftTypesField)
+    override val acceptedGiftTypes: AcceptedGiftTypes,
```

Comment:
Values without default parameters must be placed _before_ the first parameter with default value. Besides, in `acceptedGiftTypes` [ChatFullInfo](https://core.telegram.org/bots/api#chatfullinfo) placed after `permissions`. So, suitable placing would be: after `permissions` (or any first field defined above `permissions` and exists in class) if this field have no default or before the first value without defaul